CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 1.0.0 (March 8, 2017)
+
+- Switched amalgam8 sidecar to use Envoy proxy from Lyft (https://github.com/lyft/envoy)
+
+- Support for HTTP/2, gRPC proxying (downstream and upstream)
+
+- Support for gRPC over HTTP/1.1
+
+- Support for configuring circuit breakers, bulkheads, timeouts, retries
+
+- Support for different load balancing algorithms (round robin, least conn, random) ([PR#494](https://github.com/amalgam8/amalgam8/pull/494))
+
+- Deeper integration with Kubernetes: rules are now stored as third party resources ([PR#534](https://github.com/amalgam8/amalgam8/pull/534))
```

Comment:
Consider adding that can use kubectl to create rules directly and link to docs. We have two modes for k8s: with and without the rules controller. With is the preferred mode, so on second thought, maybe don't mention direct access using kubectl.